src/main/docker/docker-compose.yml

```diff
@@ -77,83 +77,274 @@ services:
+      - rabbitmq.domainId=ei-domain
+      - rabbitmq.componentName=eiffel-intelligence-artifact
+      - rabbitmq.waitlist.queue.suffix=waitlist-artifact
+      - rabbitmq.exchange.name=ei-exchange
+      - rabbitmq.user=myuser
+      - rabbitmq.password=myuser
+      - spring.data.mongodb.host=mongodb
+      - spring.data.mongodb.port=27017
+      - spring.data.mongodb.database=eiffel-intelligence-artifact
+      - missedNotificationDataBaseName=MissedNotification-artifact
+      - search.query.prefix=object
+      - aggregated.object.name=aggregatedObject
+      - spring.mail.properties.mail.smtp.auth=false
+      - spring.mail.properties.mail.smtp.starttls.enable=false
+      - er.url=http://${HOST}:8084/search/
+      - logging.level.root=OFF
+      - logging.level.org.springframework.web=ERROR
+      - logging.level.com.ericsson.ei=ERROR
+      - WAIT_MB_HOSTS=rabbitmq:15672
+      - WAIT_DB_HOSTS=mongodb:27017
+
+  ############### Stop EI backend artifact rules ##############################
+
+
+  ############### Start EI backend sourcechange rules ##########################
+  ei-backend-sourcechange:
     restart: always
-    image: eiffelericsson/eiffel-intelligence-backend:1.0.0-1
+    image: eiffelericsson/eiffel-intelligence-backend:1.0.1
     expose:
       - "8080"
     ports:
-      - "8080:8080"
+      - "8072:8080"
     depends_on:
       - mongodb
       - rabbitmq
     networks:
-          eiffel_2.0_1:
-            aliases:
-              - ei-backend
+      eiffel_2.0_1:
+        aliases:
+          - ei-backend-sourcechange
     environment:       # Overrides settings in application config file
-      - SpringApplicationName=ei-backend
+      - SpringApplicationName=ei-backend-sourcechange
       - server.port=8080
-      - rules.path=src/main/resources/ArtifactRules_new.json
+      - rules.path=/SourceChangeObjectRules.json
       - rabbitmq.host=rabbitmq
       - rabbitmq.port=5672
       - rabbitmq.domainId=ei-domain
-      - rabbitmq.componentName=ei-backend
-      - rabbitmq.waitlist.queue.suffix=waitlist
+      - rabbitmq.componentName=eiffel-intelligence-sourcechange
+      - rabbitmq.waitlist.queue.suffix=waitlist-sourcechange
       - rabbitmq.exchange.name=ei-exchange
       - rabbitmq.user=myuser
       - rabbitmq.password=myuser
       - spring.data.mongodb.host=mongodb
       - spring.data.mongodb.port=27017
-      - spring.data.mongodb.database=eiffel2_intelligence
-      - missedNotificationDataBaseName=eiffel2_intelligence_MissedNotification
+      - spring.data.mongodb.database=eiffel-intelligence-sourcechange
+      - missedNotificationDataBaseName=MissedNotification-sourcechange
+      - search.query.prefix=object
+      - aggregated.object.name=aggregatedObject
+      - spring.mail.properties.mail.smtp.auth=false
+      - spring.mail.properties.mail.smtp.starttls.enable=false
+      - er.url=http://${HOST}:8084/search/
+      - logging.level.root=OFF
+      - logging.level.org.springframework.web=ERROR
```

Comment:
I think all logging besides root logging should be default set to DEBUG.

Reply:
Fixed